src/pageScanner/helpers/helpers.js

```
@@ -7,6 +7,18 @@ export const fontSizeInPx = ( node ) => {
 	return typeof fontSize === 'number' ? fontSize : 0;
 };
 
+/**
+ * Check if an element is visibly hidden via CSS or aria attributes
+ * @param {HTMLElement} element The element to check
+ * @return {boolean} True if element is hidden
+ */
+export const isVisiblyHidden = ( element ) => {
+	const style = window.getComputedStyle( element );
+	return style.display === 'none'
+    || style.visibility === 'hidden'
```

ESLint

[error] '||' should be placed at the end of the line.

(operator-linebreak)